index.html

```diff
@@ -19,7 +19,7 @@
                 &lt;script src='stream.min.js'&gt;&lt;/script&gt;
             </pre>
 
-            <a class='download' href="stream.min.js">Download stream.js</a>
+            <a class='download' href="stream.js.zip">Download stream.js</a>
```

Comment:
Let's just host the unminified version and let the user minify it themselves then? jsDelivr sounds good regardless and we can have a minified / sourcemapped version there.